service/resource/project/resource.go

```diff
@@ -190,6 +198,10 @@ func (r *Resource) ProcessCreateState(obj, createState interface{}) error {
 		for _, p := range projectsToCreate {
 			instErr := r.installer.Install(installerspec.Project{Name: p.Name, Ref: p.Ref})
 			if instErr != nil {
+				evenErr := r.eventer.SetFailedStatus(eventerspec.DeploymentEvent{ID: 0, Name: p.Name, Sha: p.Ref})
```

Comment:
The ID has to be properly set. I need to do this in the next step. For this we need to add the deployment event ID to the TPO and wire it through the stack.